en/frames_rover/ackermann_rover.md

```diff
@@ -0,0 +1,246 @@
+# Ackermann Rover
+
+<Badge type="tip" text="main (PX4 v1.16+)" /> <Badge type="warning" text="Experimental" />
+
+An _Ackermann Rover_ controls its direction by pointing the front wheels in the direction of travel — the [ackermann steering geometry](https://en.wikipedia.org/wiki/Ackermann_steering_geometry) compensates for the fact that wheels on the inside and outside of the turn move at different rates.
+This kind of steering is used on most commercial vehicles, including cars, trucks etc.
+
+![Traxxas Stampede VXL](../../assets/airframes/rover/traxxas_stampede_vxl/stampede.jpg)
+
+::: info
+The module does not require the ackermann geometry and will work with any front-steering rover.
+:::
+
+## Airframe Setup
+
+This section will go through the most basic setup that is necessary to start using the rover.
+
+1. To enable the module flash the PX4 rover build onto your flight controller.
```

Comment:
How? QGC does offer a master "alpha" build, but not one for Rover? How do I get it and how do I install it.

If the answer is "for now you must build it yourself, we should provide build instructions for creating this firmware.

Reply:
... and we _must_ update QGC to allow selection of firmwares. Really we need a custom build server and to pull the other options from there.

Reply:
Yes, for now the rover build (will probably renamed to USV build)  has to be built by the user. I'll add a section on how to do this.
As a side note: All the new USV modules have that same problem because we can't enable them by default on most boards due to flash constraints.

> ... and we must update QGC to allow selection of firmwares. Really we need a custom build server and to pull the other options from there.

Definitely, that would make this much easier for the end user.

Reply:
Implemented in [dffb334](https://github.com/PX4/PX4-user_guide/pull/3265/commits/dffb334f2d64eabea527a27f0d1d863b39cb7927)